interfaces/builtin/joystick.go

```diff
@@ -82,15 +87,26 @@ var joystickConnectedPlugUDev = []string{
 	`KERNEL=="full", SUBSYSTEM=="mem"`,
 }
 
+type joystickInterface struct {
+	commonInterface
+}
+
+func (iface *joystickInterface) UDevConnectedPlug(spec *udev.Specification, plug *interfaces.ConnectedPlug, slot *interfaces.ConnectedSlot) error {
+	spec.TriggerSubsystem("input/joystick")
+	for _, rule := range joystickConnectedPlugUDev {
```

Comment:
I think you can just use `return iface.commonInterface.UDevConnectedPlug(spec, plug, slot)` here to _super_ call it (so to speak)